src/components/DashboardLayout/SettingsLeftNav.tsx

```diff
@@ -1,51 +1,110 @@
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import Button from 'components/Atoms/Button';
-import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faXmark, faServer, faRightToBracket } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { BrowserLocationContext } from 'data/BrowserLocationContext';
 import { RadioGroup } from '@headlessui/react';
 import { SettingsContext } from 'data/SettingsContext';
 import Avatar from 'components/Atoms/Avatar';
 import clsx from 'clsx';
 import { useUid } from 'data/UserInfo';
+import CoreSettings from 'components/Settings/CoreSettings';
+import UserSettings from 'components/Settings/UserSettings';
+
+
+export const tabList = [
+    {
+    title: 'General',
+      path: 'general',
+      content: <CoreSettings />,
+    },
+    {
+      title: 'Users',
+      path: 'users',
+      content: <UserSettings />,
+    },
+];
 
 export default function SettingsLeftNav({ className }: { className?: string }) {
+  const [setActive, setActiveTab] = useState(location.pathname.split('/')[2]);
   const { setPathname, setSearchParam } = useContext(BrowserLocationContext);
-  const { userList, selectedUser, selectUser, tabIndex } =
-    useContext(SettingsContext);
+  const { userList, selectedUser, selectUser} =
+  useContext(SettingsContext);
   const uid = useUid();
+
+  useEffect(() => {
+    setActiveTab(location.pathname.split('/')[2]);
+  }, [location.pathname]);
+
   return (
     <div className={`flex w-full flex-col items-center px-4 ${className}`}>
       <div className="flex h-full w-full grow flex-col items-start gap-4 pt-12 pb-4">
-        {selectedUser ? (
-          <Button
-            label="Unselect User"
-            icon={faXmark}
-            onClick={() => {
-              selectUser(undefined);
+      <button
+                className={clsx(
+                  'flex flex-row items-center gap-x-1.5',
+                  'cursor-pointer rounded-md py-1 px-2',
+                  'text-medium font-medium leading-5 tracking-normal',
+                  'focus-visible:outline-none focus-visible:ring-4 focus-visible:ring-blue-faded/50',
+                )}
+                onClick={() => {
+              setSearchParam('user', undefined);
+              const returnRoute = localStorage.getItem('lastVisitedRoute');
+              setPathname(returnRoute || '/');
             }}
-            className="ml-2"
-          />
-        ) : (
+        >
+        <div className='text-gray-faded/30 pr-1'>
+          <FontAwesomeIcon icon={faRightToBracket} rotation={180} size='xl' />
```

Comment:
why is there a rotation 180 lol

Reply:
for some reason font awesome only has the arrow pointing to the right so I had to 180 it lol